### PR TITLE
Extension initiation

### PR DESCRIPTION
**Pull request by Alroniks**
Task is https://oat-sa.atlassian.net/browse/TAO-8489

### CODE REVIEW

File: manifest.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+return [
+    'name' => 'taoReview',
+    'label' => 'Review',
+    'description' => 'Extension for reviewing passed tests, with the display of actual and correct answers.',
+    'license' => 'GPL-2.0',
+    'version' => '0.1.0',
+    'author' => 'Open Assessment Technologies SA',
+    'requires' => [
+        'tao' => '*',
```
Comment (Skyld) [inline manifest.php]: Are we sure about this?

File: views/build/grunt/bundle.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+/**
+ * Configure the extension bundles
+ * @author Ivan Klimchuk <ivan@taotesting.com>
+ */
+module.exports = function(grunt) {
+    'use strict';
+
+    grunt.config.merge({
+        bundle : {
+            taobosaselor : {
```
Comment (Skyld) [inline views/build/grunt/bundle.js]: Shouldn't this extension be client agnostic?

File: manifest.php (hunk 1 above)
Comment (Alroniks) [reply manifest.php]: While we don't have specific things that require a specific version, I guess it makes sense.

File: views/build/grunt/bundle.js (hunk 2 above)
Comment (Alroniks) [reply views/build/grunt/bundle.js]: Copy-pasted part, fixed.

**Pull request merged**